### PR TITLE
Inital setup with apollo

### PR DESCRIPTION
**Pull request by tony-matheus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "graphql-import": "^1.0.2",
+    "graphql-import-node": "^0.0.4",
+    "http": "^0.0.1-security",
+    "ncp": "^2.0.0"
+  },
+  "devDependencies": {
+    "nodemon": "^2.0.4",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.9.7"
+  }
```
[the remark this answers is not in the document]
Comment (tony-matheus) [reply package.json:36]: should i import everything again? or is this just a tip for future cases?

[the remark this answers is not in the document]
Comment (ItaloSampaio) [reply package.json:36]: After editing all dependencies in `package.json`, just run `yarn` and fix what is broken

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "graphql-depth-limit": "1.1.0",
+    "graphql-import": "1.0.2",
+    "graphql-import-node": "0.0.4",
+    "http": "0.0.1-security",
+    "ncp": "2.0.0"
+  },
+  "devDependencies": {
+    "nodemon": "^2.0.4",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.9.7"
```
Comment (ItaloSampaio) [inline package.json:35]: You should pin dev dependencies too

Comment (tony-matheus) [reply package.json:35]: thanks, i just did

**Pull request merged**